### DISCUSSION TITLE
Show post author

### DISCUSSION THREAD

**Discussion by scott-ease** [xenocrat/chyrp-lite#194] [General]
Is there a way to show post author with link to all posts of that specific author? I think that's a basic function, just like clicking category will show all posts of that specific category. Thank you.

**Comment by xenocrat** [comment]
Hello there,

You can show all posts by an author using the search function. Enter "author:username" in the search box, e.g. "author:scott". But I agree there should be a built-in route to easily view all posts by an author; I will add this.
